File: Libraries/Old/Java/Node.java
import java.util.ArrayList;

public class Node extends EventTarget {
    public final String baseURI = "";
    public final ArrayList<Node> childNodes = new ArrayList<Node>();
    public final Node firstChild = null;
    public final boolean isConnected = false;
    public final Node lastChild = null;
    public final Node nextSibling = null;
    public final String nodeName = "";
    public final NodeType nodeType;
    public String nodeValue = new String();
    public final Node ownerDocument = null;
    public final Node parentNode = null;
    public final Node parentElement = null;
    public final Node previousSibling = null;
    public String textContent = new String();

    public Node(NodeType nodetype) {
        this.nodeType = nodetype;
    }

    public final void appendChild(Node childNode) {}
    public final Node cloneNode() {
        return this;
    }
    public final void compareDocumentPosition(Node other) {}
    public final boolean contains(Node node) {
        for (Node i : this.childNodes) {
            if (i.equals(node)) return true;
        }
        return false;
    }
    public final Node getRootNode() {
        return this.parentElement;
    }
    public final boolean hasChildNodes() {
        if (this.childNodes.size() > 0) return true;
        else return false;
    }
    public final void insertBefore(Node node) {}
    public final boolean isDefaultNamespace() {
        if (this.baseURI == this.ownerDocument.baseURI) return true;
        else return false;
    }
    public final boolean isEqualNode(Node other) {
        return this.equals(other);
    }
    public final boolean isSameNode(Node other) {
        return this == other;
    }
    public final String lookupNamespace() {
        return this.baseURI;
    }
    public final void normalize() {}
    public final void removeChild(Node child) {}
    public final void replaceChild(Node child, Node newChild) {}
}

File: Libraries/Old/Kotlin/Node.kt
package devkit.web.interfaces

import kotlin.collections.*

open public class Node: EventTarget() {
    private var baseURI: String = ""
    private val childNodes: MutableList<Node> = mutableListOf()
    private var firstChild: Node? = null
    private var isConnected: Boolean = null
    private var lastChild: Node? = null
    private var nextSibling: Node? = null
    private var nodeName: String = ""
    private var nodeType: Int = NodeTypes.UNKNOWN_NODE
    public var nodeValue: String = ""
    private var parentNode: Node? = null
    private var parentElement: Element? = null
    private var previousSibling: Node? = null
    private var textContent: String = ""

    public fun appendChild(childNode: Node): Unit {}
    public fun cloneNode(): Node {}
    public fun compareDocumentPosition(other: Node): Int {}
    public fun contains(node: Node): Boolean {}
    public fun getRootNode(): Node {}
    public fun hasChildNodes(nodes: Array<Node>): Boolean {}
    public fun insertBefore(node: Node): Unit {}
    public fun isDefaultNamespace(namespaceURI: String): Boolean {}
    public fun isEqualNode(node: Node): Boolean {}
    public fun isSameNode(node: Node): Boolean {}
    public fun lookupPrefix(prefix: String): Unit {}
    public fun lookupNamespaceURI(uri: String): Unit {}
    public fun normalize(): Unit {}
    public fun removeChild(child: Node): Unit {}
    public fun replaceChild(old: Node, new: Node): Unit {}
}